source/includes/php/restResponse.md

```diff
@@ -0,0 +1,53 @@
+# RestResponse
+
+> Get a successful RestResponse
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+\Shoprunback\RestClient::getClient()->setToken('yourApiToken');
+
+try {
+  $response = $restClient->request(
+    \Shoprunback\Elements\Product::showEndpoint('1f27f9d9-3b5c-4152-98b7-760f56967dea'),
+    \Shoprunback\RestClient::GET
+  );
+} catch(RestClientError $e) {
+
+}
+
+$response->getCode(); // Returns the HTTP code
+$response->getBody(); // Returns the result of the API call (Here it returns the product with the id '1f27f9d9-3b5c-4152-98b7-760f56967dea')
+```
+
+> Get a failed RestResponse
+
+```php
+<?php
+require 'path/to/lib/shoprunback-php/init.php';
+
+\Shoprunback\RestClient::getClient()->setToken('yourApiToken');
+
+try {
+  $product = \Shoprunback\Elements\Product::retrieve('a wrong ID');
+} catch (\Shoprunback\Error\Error $e) {
+  if (is_a($e, \Shoprunback\Error\RestClientError)) {
+    $response = $e->response;
+
+    $response->getCode(); // Returns the HTTP code
+    $response->getErrors(); // Returns the errors
+  }
+}
+```
+
+The RestResponse contains the **response of an API call**.
+
+The **successful RestResponses** are **managed in the library**. If you **meet a case where you need to directly use a successful RestResponse**, please **send an email** to tech_at_shoprunback.com.
```

Comment:
Not a big fan of this. This should not happen. So just remove those 2 lines. If you put the doubt in someone's mind, they will eventually doubt and might end up asking unrelated questions.